docs/_ext/single_sourced_data.py

```diff
@@ -45,8 +45,8 @@
 ]
 
 
-DOCS_DIR = Path(__file__).parents[1].resolve()
-PROJECT_DIR = DOCS_DIR.parent
+DOCS_DIR = Path(__file__).resolve().parents[1]
+PROJECT_DIR = Path(__file__).resolve().parents[2]
```

Comment:
I don't like that it's copy-paste — I specifically tried to avoid it. Otherwise, I would inline this in places where it's being used :man_shrugging: